imapclient/imapclient.py

```diff
@@ -1107,7 +1108,21 @@ def expunge(self):
 
         See :rfc:`3501#section-6.4.3` section 6.4.3 and
         :rfc:`3501#section-7.4.1` section 7.4.1 for more details.
+
+        When *messages* are specified, remove the specified messages
+        from the selected folder, provided those messages also have
+        the ``\\Deleted`` flag set. The return value is ``None`` in
+        this case.
+
+        Expunging messages by id(s) requires that *use_uid* is
+        ``True`` for the client.
+
+        See :rfc:`4315#section-2.1` section 2.1 for more details.
         """
+        if messages:
+          if not self.use_uid:
```

Comment:
Could you indent with four spaces?

Reply:
done, rev# b566947e46dd5699f66c12507e383f7578f36942